### PR TITLE
submitting project one

### PR DESCRIPTION
**Pull request by bmdh05**
bryan.diaz@lc.cuny.edu

### CODE REVIEW

File: project_one/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  </form>
+  
+
+<script type = "text/javascript">
+
+    function addingText() {
+        
+        var someText = document.createElement("P");
+        someText.innerText = "consequence";
+        document.getElementById("grading-element-one-consequence").appendChild(someText);
```
Comment (lehmanclass) [inline project_one/index.html]: Very close. Give `someText` the id `grading-element-one-consequence` via javascript and append it to an element.

File: project_one/index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>project one</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+</head>
+<body>
+    <a id= "grading-element-two" href ="about.html">About</a>
```
Comment (lehmanclass) [inline project_one/index.html]: great, works as expected.

File: project_one/index.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    <a href = "index.html">Home</a>
+    <div id= "heading">
+        <h1>Project One</h1>
+
+    </div>
+    <br>
+    <br>
+    <button id="grading-element-one" onclick="addingText()">Your DOM will suffer consequences</button>
+
+    <div id= "grading-element-one-consequence">
```
Comment (lehmanclass) [inline project_one/index.html]: should not be in the `index.html`...create & attach to the dom when `addingText()` fires, please.

File: project_one/index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        
+        
+    
+    </div>
+    
+    <br>
+    <br>
+    <button id= "grading-element-three" onclick="showing()">Show the ninja div</button>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected!

File: project_one/index.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    document.getElementById("grading-element-three-consequence").style.visibility = "visible" ;
+
+}
+
+function checkedForm(){
+    var x = document.forms["grading-element-four"]["testOne"].value;
+    var y = document.forms["grading-element-four"]["testTwo"].value;
+    var z = document.forms["grading-element-four"]["testThree"].value;
+    
+  if (x == "" ||  y == "" || z == "") {
```
Comment (lehmanclass) [inline project_one/index.html]: validation is great! But I want _your javascript_ to put values in the three fields `testOne`, `testTwo`, `testThree`, not force me to do it.

File: project_one/index.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    <button id= "grading-element-three" onclick="showing()">Show the ninja div</button>
+    <br>
+    <div id ="grading-element-three-consequence">
+
+        <p>the div ninja was discover, is not a good ninja :(</p>
+        
+    </div>
+
+    
+   <div id ="grading-element-five">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected!

File: project_one/index.html (hunk 5 above)
Comment (bmdh05) [reply project_one/index.html]: can I add an id to the fields to set the values.

**Pull request closed without merging**